### PR TITLE
Align RFC 1008 with recent decisions on TLS

### PR DESCRIPTION
**Pull request by fantix**
~Pending:~

- [ ] ~rename `edgedb authenticate`~

### CODE REVIEW

**Review by elprans — APPROVED** [review]

**Comment by 1st1** [comment]
@fantix Can we also change `edgedb authenticate` to `edgedb instance login` / `edgedb instance logout` in a follow up PR?

**Comment by fantix** [comment]
Oh sure - I can get this one merged first then.

File: text/1008-tls-and-alpn.rst [hunk 1]
```diff
@@ -149,107 +150,105 @@ Client-side Server Certificate Verification
...
+certificate verification should always be enabled. By default, the
+system-wide trusted CA certificates are usually used to verify server
+certificates. For server certificates that are signed by untrusted CA,
+the users could provide the path to the specific CA certificate file
+they trust - for CLI the option is ``--tls-ca-file``, for language
+bindings the option is usually ``tls_ca_file`` or ``tlsCaFile``.
+
+In order to accept the self-signed certificate, at the time of
+certificate generation, the EdgeDB CLI will also copy the generated
+certificate into the so-called ``credentials.json`` - a group of JSON
```
Comment (1st1) [inline text/1008-tls-and-alpn.rst:162]: We already have `/home/yury/.config/edgedb/credentials/foo.json` --- when you say `credentials.json` do you mean `credentials/<instance-name>.json`?

File: text/1008-tls-and-alpn.rst [hunk 2]
```diff
@@ -149,107 +150,105 @@ Client-side Server Certificate Verification
...
-file on the server-side, a new command is proposed to create a local
-``credentials.json`` file for all future connections to the same
-instance::
+value of ``tls_cert_data`` and trust only that certificate for
+connecting to the EdgeDB instance if ``tls_cert_data`` is present.
+
+The client allows the user to decide if hostname should be checked. For
+CLI, the options are ``--tls-verify-hostname`` to enable the check, and
+``--no-tls-verify-hostname`` to disable it. For language bindings, the
+option is usually a bool ``tls_verify_hostname``, where ``true`` means
```
Comment (1st1) [inline text/1008-tls-and-alpn.rst:183]: nit:

```suggestion
option is usually a boolean ``tls_verify_hostname``, where ``true`` means
```

**Review by 1st1 — APPROVED** [review]

File: text/1008-tls-and-alpn.rst (hunk 1 above)
Comment (fantix) [reply text/1008-tls-and-alpn.rst:162]: Yes, I should probably remove the double quotes for inline code block style, and refer to it as just "credentials JSON files"

**Pull request merged**